test/decoders/test_video_decoder_ops.py

```diff
@@ -198,6 +222,40 @@ def test_get_frames_by_pts(self):
         with pytest.raises(AssertionError):
             assert_tensor_equal(frames[0], frames[-1])
 
+    # TODO: Figure out how to parameterize this test to run on both CPU and CUDA.abs
+    # The question is how to have the @needs_cuda decorator with the pytest.mark.parametrize
+    # decorator on the same test.
```

Comment:
It's simple!

We just need to define this new util

https://github.com/pytorch/vision/blob/e9a3213524a0abd609ac7330cf170b9e19917d39/test/common_utils.py#L122-L125

and it can be used like this

https://github.com/pytorch/vision/blob/e9a3213524a0abd609ac7330cf170b9e19917d39/test/test_utils.py#L221


If you want, we can merge this PR as-is and follow-up with that